app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/ToolbarViewModel.kt

```diff
@@ -70,7 +70,7 @@ class ToolbarViewModel(
 
     @UiThread
     fun backButtonClicked() {
-        sessionRepo.exitFullScreenIfPossibleAndBack()
+        sessionRepo.attemptBack(forceYouTubeExit = true)
```

Comment:
What if the user wants to go back through the YT history stack? e.g. they have watched a series of videos. (Though maybe this is already the behavior now...)
But this is a weird case - e.g. we want to handle web browsing history, but not trigger things like the "back" cursor movement.
What do you think?

Reply:
I _think_ that's handled by YouTube using the ESC key, but I'll verify that before merging.

Reply:
> I think that's handled by YouTube using the ESC key, but I'll verify that before merging.

It looks like this is true.  It is interesting to note that, if you've watched a bunch of videos, back will still just take you to YouTube TV home.  But that's really Google's decision.